### PR TITLE
feat: import and handle dataviz tokens

### PR DESCRIPTION
**Pull request by BalbinaK**
## Description

Fixes [WARP-644](https://nmp-jira.atlassian.net/browse/WARP-644)

This PR adds logic to import DataViz tokens, transform them and generate assets for web, iOS and Android using StyleDictionary. The iOS and Android part has not been looked at, but the colors are generated and zipped.
The build script was refactored to cover both DataViz and Warp tokens and most logic was stored in `buildUtils.js`.

## Testing
Run `pnpm build:dataviz` (you will need the Figma token) and check the `/output` folder for generated assets.

### CODE REVIEW

File: buildUtils.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  outputFilePath,
+  tokenVariableCollection,
+  isDataviz = false,
+}) {
+  spinner.start("Reading Figma access token");
+
+  let token = process.env.FIGMA_TOKEN;
+
+  if (token) {
+    spinner.succeed("Using Figma access token from environment viariables");
```
Comment (felicia-haggqvist) [inline buildUtils.js]: found a tiny typo :) 
```suggestion
    spinner.succeed("Using Figma access token from environment variables");
```

**Review by felicia-haggqvist — APPROVED** [review]
Really nice work! 🥳 🎉

**Review by imprashast — APPROVED** [review]
Good one. Nice refactors 👍

**Comment by BalbinaK** [comment]
Before I merge I think I need to revisit the build scripts so I can add the dataviz tokens fetching and release to the github workflows.

**Comment by BalbinaK** [comment]
@imprashast before I merge, I'd like you to have a look at the last commit: [Add workflows for fetching Figma tokens and update structure](https://github.com/warp-ds/tokens/pull/13/commits/0ede0ff49ff4750a4652135477d60faf69405d50) 
[0ede0ff](https://github.com/warp-ds/tokens/pull/13/commits/0ede0ff49ff4750a4652135477d60faf69405d50)
Changes included there:
- Create a new GitHub Action to fetch dataviz tokens from Figma.
- Rename existing workflow to focus on fetching warp tokens.
- Update import scripts for dataviz and warp tokens.
- Only build assets in the release workflow

**Pull request merged**